### ISSUE TITLE
Client [/move_group] wants topic /multi_dof_joint_trajectory_action/feedback to have datatype/md5sum [action_controller/MultiDofFollowJointTrajectoryActionFeedback

### ISSUE THREAD

**Issue by boonflies** [wilselby/ROS_quadrotor_simulator#29]
When I try to launch,
**roslaunch quad_3dnav quad_3dnav.launch**
I receive the following error.

ERROR [/tmp/binarydeb/ros-indigo-roscpp-1.11.21/src/libros/publication.cpp:498(Publication::validateHeader) [topics: /rosout, /cmd_vel, /cmd_3dnav, /multi_dof_joint_trajectory_action/result, /multi_dof_joint_trajectory_action/feedback, /multi_dof_joint_trajectory_action/status] Client [/move_group] wants topic /multi_dof_joint_trajectory_action/feedback to have datatype/md5sum [action_controller/MultiDofFollowJointTrajectoryActionFeedback/bb90301ebf9ac17d9c3bcafa7a141cba], but our version has [action_controller/MultiDofFollowJointTrajectoryActionFeedback/0611ea4a62fe3c6e8470a86482016c70]. Dropping connection.
1.425000000 ERROR [/tmp/binarydeb/ros-indigo-roscpp-1.11.21/src/libros/publication.cpp:498(Publication::validateHeader) [topics: /rosout, /move_group/planning_scene_monitor/parameter_descriptions, /move_group/planning_scene_monitor/parameter_updates, /move_group/monitored_planning_scene, /move_group/ompl/parameter_descriptions, /move_group/ompl/parameter_updates, /move_group/display_planned_path, /move_group/display_contacts, /multi_dof_joint_trajectory_action/goal, /multi_dof_joint_trajectory_action/cancel] Client [/action_controller] wants topic /multi_dof_joint_trajectory_action/goal to have datatype/md5sum [action_controller/MultiDofFollowJointTrajectoryActionGoal/762be92da0a19418188f1bfa9dab9fe5], but our version has [action_controller/MultiDofFollowJointTrajectoryActionGoal/1292bbe002476592a0f3166376533ba1].

**Comment by wilselby** [comment]
Try updating your packages.

**Comment by tahsinkose** [comment]
I have posted in @wilselby's  blog as well. If you are using Kinetic like me for to get this project up and running, probably update&upgrade sequence won't suffice to get rid of this error. Instead, reading a little bit documentation and migration stuff enlightens the issue. 

Actually, the problem is related to [custom action_controller package](https://github.com/AlessioTonioni/Autonomous-Flight-ROS/tree/master/ROS%20-%20Indigo), which @wilselby has also a copy of it in its project repository. They should not be blamed, since MoveIt! indeed has a `moveit_msgs/MultiDOFJointTrajectory.msg` file for ROS Indigo. However, since Kinetic, they migrate trajectory stuff under `trajectory_msgs` package. That's why, at least in Kinetic case, the identical run-time error occurs. Changing every occurence of `moveit_msgs` with `trajectory_msgs` in the file `MultiDofFollowJointTrajectory.action` of `action_controller` package and having a brand new compilation would resolve this problem (Also don't forget to remove the contents of `msg_gen` and `build` directories. Since CMake checks for diff during compilation, they may not be regenerated and changes may not be reflected.)

EDIT: I was following the tutorial at @wilselby's website, and the link he provided has an old `action_controller`. Contrarily, this repository stores the updated version. So, the changes I described above are already implemented.
